packages/custom-elements/ts_src/CustomElementInternals.ts

```diff
@@ -280,8 +280,8 @@ export default class CustomElementInternals {
       if (definition) {
         this._upgradeAnElement(element, definition);
       }
-    } catch (e) {
-      this.reportTheException(e);
+    } catch (e: unknown) {
+      this.reportTheException(e as Error);
```

Comment:
We have a rule that prohibits `any`, so I've used `unknown` here and then cast to `Error`. `reportTheException` should probably take `unknown` instead.

Reply:
`reportTheException` now takes unknown.